Bugfix CustomerImporter.php

micronax opened the pull request.

swDennis commented on Components/DbServices/Import/CustomerImporter.php:
```diff
@@ -782,9 +782,8 @@ private function newsletterSubscribe(array $customer)
-                        VALUES (1, {$customer['newslettergroupID']}, {$customer['email']});";
-                $this->db->query($sql);
+                $sql = 'INSERT INTO s_campaigns_mailaddresses (customer, groupID, email)
+                VALUES (1, ' . $customer['newslettergroupID'] . ',"' . $customer['email'] . '");';
```
The SQL query is nowhere executed

micronax commented:
Shoot! I forgot to paste the `$this->db->query($sql);` statement. I added it in a separate commit.

The bug I fixed is documented in the official shopware forum: 

https://forum.shopware.com/discussion/41654/migration-von-prestashop-probleme-beim-kundenimport-sql-syntax-near-gmx-de

micronax commented:
Siehe auch: https://issues.shopware.com/issues/PT-7540

mitelg commented:
hey @micronax 
unfortunately the current state of your PR has a conflict. is it still relevant? if so, could you please rebase your PR to the current master?
sorry for the late response! :pray:

mitelg commented:
hey @micronax 

Sorry for the late answer!
The corresponding ticket was closed. If this change is still relevant for you, please feel free to create a new PR.
